packages/kit/src/views/Send/components/SendFee/FeeEditor.tsx

```
@@ -1,11 +1,11 @@
 import { useCallback, useEffect, useMemo, useState } from 'react';
 
 import BigNumber from 'bignumber.js';
-import { isNaN, isNil, isNumber } from 'lodash';
+import { isNaN, isNil } from 'lodash';
```

Biome

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)